dracut/modules.d/99asahi-firmware/load-asahi-firmware.sh

```diff
@@ -43,5 +43,5 @@ if [ ! -e "$VENDORFW/firmware.cpio" ]; then
     return 1
 fi
 
-( cd /; cpio -i < "$VENDORFW/firmware.cpio" )
+( cd /; cpio --quiet -i < "$VENDORFW/firmware.cpio" )
```

Comment:
If we're running in debug mode, we probably want it to not be quiet.

Reply:
I don't think this one matters, all it does is suppress the "XXX blocks" line, which isn't terribly useful either day.

Reply:
Agreed.